Translation/Extractor/File/FormExtractor.php

```diff
@@ -119,7 +120,24 @@ public function enterNode(Node $node)
                 }
 
                 if ('choices' === $item->key->value) {
+
+                    //Checking for the choice_as_values in the same form item
+                    $choices_as_values = false;
+                    foreach ($node->items as $cv_item) {
+                        if ($cv_item->key !== null && 'choices_as_values' === $cv_item->key->value) {
+                            $choices_as_values = ($cv_item->value->name->parts[0] === 'true');
+                        }
+                    }
+
                     foreach ($item->value->items as $sitem) {
+                        // If we have a choice as value that differ from the Symfony default strategy
+                        // we should invert the key and the value
+                        if (Kernel::VERSION_ID < 30000 && $choices_as_values === true || Kernel::VERSION_ID > 30000 && $choices_as_values === false ) {
```

Comment:
This looks a bit strange. The second choices_as_values should be removed, right?